### PR TITLE
Add containerized build / test

### PR DESCRIPTION
**Pull request by caseydavenport**
Useful for running tests on a Mac!

### CODE REVIEW

**Comment by gunjan5** [comment]
It fails with this:
```bash
Makefile:14: recipe for target 'ut' failed
make: *** [ut] Error 1
make: *** [test-containerized] Error 2
```

Adding `RUN go get github.com/onsi/gomega` to the `Dockerfile.build` fixes it

File: Dockerfile.build [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  apk add --allow-untrusted glibc.apk && \
+  curl -o glibc-bin.apk -L "https://github.com/andyshinn/alpine-pkg-glibc/releases/download/${GLIBC_VERSION}/glibc-bin-${GLIBC_VERSION}.apk" && \
+  apk add --allow-untrusted glibc-bin.apk && \
+  /usr/glibc-compat/sbin/ldconfig /lib /usr/glibc/usr/lib && \
+  echo 'hosts: files mdns4_minimal [NOTFOUND=return] dns mdns4' >> /etc/nsswitch.conf && \
+  apk del curl && \
+  rm -f glibc.apk glibc-bin.apk && \
+  rm -rf /var/cache/apk/*
+
+RUN go get github.com/onsi/ginkgo/ginkgo
```
Comment (gunjan5) [inline Dockerfile.build]: add `RUN go get github.com/onsi/gomega` here

**Comment by caseydavenport** [comment]
Hm... works fine for me.  I'll investigate.

**Comment by caseydavenport** [comment]
@robbrockbank could you give this guy a look?

**Comment by robbrockbank** [comment]
LGTM

**Pull request merged**